src/webparts/todolist/components/ToDoList.tsx

```diff
@@ -167,7 +115,7 @@ const ItemList: React.FC<IItemListProps> = (props: IItemListProps) => {
 
   React.useEffect(() => {
     getListData();
```

Comment:
Will be better to put props.listName as a parameter in method. Let's make function more generic